exchanges/mock/server_test.go

```diff
@@ -38,89 +37,54 @@ func TestNewVCRServer(t *testing.T) {
 		Amount:   1,
 		Currency: "bitcoin",
 	})
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "json.Marshal must not error")
 
 	testValue := HTTPResponse{Data: rp, QueryString: queryString, BodyParams: queryString}
 	test1.Routes["/test"][http.MethodGet] = []HTTPResponse{testValue}
 
 	payload, err := json.Marshal(test1)
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "json.Marshal must not error")
 
 	err = os.WriteFile(testFile, payload, os.ModePerm)
-	if err != nil {
-		t.Fatal("marshal error", err)
-	}
+	require.NoError(t, err, "json.Marshal must not error")
 
 	deets, client, err := NewVCRServer(testFile)
-	if err != nil {
-		t.Error("NewVCRServer error", err)
-	}
+	assert.NoError(t, err, "NewVCRServer should not error")
 
 	err = common.SetHTTPClient(client) // Set common package global HTTP Client
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	_, err = common.SendHTTPRequest(t.Context(),
 		http.MethodGet,
 		"http://localhost:300/somethingElse?"+queryString,
 		nil,
 		bytes.NewBufferString(""), true)
-	if err == nil {
-		t.Error("Sending http request expected an error")
-	}
+	assert.Error(t, err, "Sending http request expected an error")
 
 	// Expected good outcome
 	r, err := common.SendHTTPRequest(t.Context(),
 		http.MethodGet,
 		deets,
 		nil,
 		bytes.NewBufferString(""), true)
-	if err != nil {
-		t.Error("Sending http request error", err)
-	}
-
-	if !strings.Contains(string(r), "404 page not found") {
-		t.Error("Was not expecting any value returned:", r)
-	}
+	assert.NoErrorf(t, err, "Sending http request error: %v", err)
+	assert.Containsf(t, string(r), "404 page not found", "Was not expecting any value returned: %s", string(r))
```

Comment:
🚧 This error message does not make sense with assertify.
```suggestion
	assert.Containsf(t, string(r), "404 page not found", "SendHTTPRequest return should only contain 404")
```